lib/rspec/rails/matchers/action_cable.rb

```diff
@@ -45,12 +45,17 @@ module ActionCable
       #     expect {
       #       ActionCable.server.broadcast "messages", text: 'Hi!'
       #     }.to have_broadcasted_to("messages").with(text: 'Hi!')
+      extend RSpec::Matchers::DSL
```

Comment:
This needs to moved to the top of the module, under the module doc comment, so that yard works properly.